### PR TITLE
Bujv 8

### PR DESCRIPTION
**Pull request by Shreyaswrp**

### CODE REVIEW

File: src/AuthorCard/AuthorCard.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<template>
+  <Card>
+    <template #header>
+      <img src="profilePicture" alt="John" style="width: 100%" />
+    </template>
```
Comment (paiameya) [inline src/AuthorCard/AuthorCard.vue]: You need to use the Avatar component created by Harshal to handle this

File: src/AuthorCard/AuthorCard.vue [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <template #title>
+      {{ name }}
+    </template>
+    <template #content> CEO & Founder, Example </template>
```
Comment (paiameya) [inline src/AuthorCard/AuthorCard.vue]: Don't push dummy data

File: src/AuthorCard/AuthorCard.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    }
+  },
+}
+</script>
+
+<style scoped>
+.p-card {
+  width: 310px;
+  text-align: center;
+  padding: 27px 21px 12px 21px;
```
Comment (paiameya) [inline src/AuthorCard/AuthorCard.vue]: Please use em/rem instead of px

**Review by paiameya — CHANGES_REQUESTED** [review]

File: src/AuthorCard/AuthorCard.vue (hunk 1 above)
Comment (Shreyaswrp) [reply src/AuthorCard/AuthorCard.vue]: after you deploy, I will use his component.

**Review by paiameya — APPROVED** [review]

**Pull request merged**